shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -1358,11 +1377,18 @@ public DisplayFeature(Rect bounds, DisplayFeatureType type, DisplayFeatureState
       this.type = type;
       this.state = state;
     }
+  }
 
-    public DisplayFeature(Rect bounds, DisplayFeatureType type) {
+  /**
+   * Description of a cutout on the physical display.
+   *
+   * <p>A simplified analog to {@link DisplayFeature} that specifically handles display cutouts.
+   */
+  public static final class DisplayCutout {
+    public final Rect bounds;
+
+    public DisplayCutout(Rect bounds) {
```

Comment:
should this default to unknown? (or maybe a new state value?

Reply:
I'm not clear on what you are asking here. Should what default to unknown?

Reply:
The Display feature class which we previously used for cutouts has a type and a state.  Should those be hardcoded in the DisplayCutout class as type and state then let the code above in setViewportMetrics refer to DisplayCutout.type and DisplayCutout.state instead of using for example DisplayFeatureState.UNKNOWN.encodedValue.

Reply:
I do not have any strong opinions on it, so I will change it accordingly.